data/simple-icons.json

```
@@ -11382,6 +11382,20 @@
 		"source": "https://en.facebookbrand.com/oculus/assets/oculus?audience=oculus-landing",
 		"guidelines": "https://en.facebookbrand.com/oculus/"
 	},
+	{
+		"title": "Odido",
+		"hex": "000000",
+		"source": "https://www.odido.nl/Assets/Lfwd/Static/odido-logo-black.svg",
+		"guidelines": "https://merk.odido.nl/en/logo",
```

We should link to the full brand guidelines here.
```suggestion
		"guidelines": "https://merk.odido.nl/en",
```